spec/requests/posts_spec.rb

```diff
@@ -1,17 +1,37 @@
 require 'rails_helper'
 
 RSpec.describe 'Posts', type: :request do
-  describe 'GET /index' do
+  describe 'GET /users/:id/posts' do
     it 'returns http success' do
-      get '/posts/index'
+      get '/users/1/posts'
       expect(response).to have_http_status(:success)
     end
+
+    it 'it renders index template' do
+      get '/users/1/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'it response body includes correct placeholder text.' do
+      get '/users/1/posts'
+      expect(response.body).to include('This a page to display posts')
+    end
   end
 
-  describe 'GET /show' do
+  describe 'GET /users/:id/posts/:id' do
     it 'returns http success' do
-      get '/posts/show'
+      get '/users/1/posts/1'
       expect(response).to have_http_status(:success)
     end
   end
+
```

Comment:
You made a great effort! I liked your approach. 
[OPTIONAL] However, I would suggest you do one more step to make your code much better! I would suggest cleaning your unused test from your file. Other than that your project is good.

<img width="1048" alt="Screen Shot 2022-08-23 at 20 04 50" src="https://user-images.githubusercontent.com/69011963/186194006-a014f6a9-2f03-42a1-bce2-bd61ae55e15d.png">